### PR TITLE
Adding swedish translation for SponsorBlock & Extras

### PR DESCRIPTION
**Pull request by Mr-Bajs**
Adding missing swedish translation for the sponsorblock segment

### CODE REVIEW

File: app/src/main/res/values-sv/strings.xml [hunk 1]
```diff
@@ -763,4 +763,40 @@
...
+    <string name="sponsor_block_enable_title">Hoppa över sponsorer</string>
+    <string name="sponsor_block_enable_summary">Använd Sponsorblocks API för att automatiskt hoppa över sponsorer i videor. Detta fungerar närvarande bara för Youtubevidor.</string>
+    <string name="sponsor_block_api_url_title">API-Url</string>
+    <string name="sponsor_block_api_url_summary"></string>
+    <string name="sponsor_block_notifications_title">Avisera när sponsorer hoppas över</string>
+    <string name="sponsor_block_notifications_summary">Visa en avisering när en sponsor hoppas över automatiskt</string>
+    <string name="sponsor_block_privacy_title">Visa Integritetspolicy</string>
+    <string name="sponsor_block_privacy_summary">Visa SponsorBlocks Integritetspolicy</string>
+    <string name="sponsor_block_api_url_help_text">Det är denna URL som kommer förfrågas när applikationen behöver veta vilka delar av en video som skall hoppas över.\n\nDu kan välja den officiela URL genom att klicka "Use Official"-alternativet nedanför. men det rekomenderas starkt att du läser SponsorBlocks integritetspolicy innan du gör det.</string>
```
Comment (Atemu) [inline app/src/main/res/values-sv/strings.xml:778]: > klicka "Use Official"-alternativet

We should probably make that string translatable.

File: app/src/main/res/values-sv/strings.xml [hunk 2]
```diff
@@ -763,4 +763,40 @@
...
-</resources>
\ No newline at end of file
+        <!-- SponsorBlock -->
+    <string name="sponsor_block">SponsorBlock</string>
+    <string name="sponsor_block_home_page_title">Visa Hemsida</string>
+    <string name="sponsor_block_home_page_summary">Visa SponsorBlocks officiella hemsida.</string>
+    <string name="sponsor_block_enable_title">Hoppa över sponsorer</string>
+    <string name="sponsor_block_enable_summary">Använd Sponsorblocks API för att automatiskt hoppa över sponsorer i videor. Detta fungerar närvarande bara för Youtubevidor.</string>
+    <string name="sponsor_block_api_url_title">API-Url</string>
+    <string name="sponsor_block_api_url_summary"></string>
```
Comment (Atemu) [inline app/src/main/res/values-sv/strings.xml]: What's up with the summary?

**Review by Atemu — COMMENTED** [review]
I don't speak Swedish but German is close enough to tell these translations don't [say something entirely different](https://www.phoronix.com/news/Ubuntu-23.10-Recalled).

File: app/src/main/res/values-sv/strings.xml (hunk 2 above)
Comment (Mr-Bajs) [reply app/src/main/res/values-sv/strings.xml]: Missed that one.

File: app/src/main/res/values-sv/strings.xml (hunk 1 above)
Comment (Mr-Bajs) [reply app/src/main/res/values-sv/strings.xml:778]: Absolutely.

File: app/src/main/res/values-sv/strings.xml [hunk 3]
```diff
@@ -763,4 +763,52 @@
...
+    <string name="enable_local_player_title">Aktivera Lokal Spelare (alpha)</string>
+    <string name="enable_local_player_summary">Använd inbyggd spelare för lokal uppspelning. Detta är fortfarande tidigt i utvecklingen så det kommer troligen uppstå många fel, inklusive problem med existerade spelare.</string>
+    <string name="force_auto_fullscreen_title">Tvinga Automatisk Fullskärm</string>
+    <string name="force_auto_fullscreen_summary">Om aktiverat så kommer fullskärm tvingas om enheten är i landskapsläge, även om enheten är en TV eller padda.</string>
+    <string name="disable_error_reports_title">Inaktivera felrapporter</string>
+    <string name="disable_error_reports_summary">Förhindrar att alla felmeddelanden visas. Det kan resultera att appen beter sig märkligt. <b>ANVÄND PÅ EGEN RISK!</b></string>
+    <string name="enable_return_youtube_dislike_title">Visa antal ogillningar</string>
+    <string name="enable_return_youtube_dislike_summary">Använd ReturnYouTubeDislikes API  för att visa antal ogillningar för en video. Funderar bara för Youtubevideor.\n<b>VARNING: Din IP-adress kommer bli syndlig för API:n. Används på egen risk!</b></string>
```
Comment (Atemu) [inline app/src/main/res/values-sv/strings.xml]: Is the double space intended?

```suggestion
    <string name="enable_return_youtube_dislike_summary">Använd ReturnYouTubeDislikes API för att visa antal ogillningar för en video. Funderar bara för Youtubevideor.\n<b>VARNING: Din IP-adress kommer bli syndlig för API:n. Används på egen risk!</b></string>
```

Comment (Mr-Bajs) [reply app/src/main/res/values-sv/strings.xml]: fixed

**Pull request closed without merging**